shared/fs/sortbar/container.js

```diff
@@ -30,12 +31,9 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
       navigateAppend([
         {
           props: {
-            sortSettingToAction: (sortSetting: Types.SortSetting) => (evt?: SyntheticEvent<>) => {
+            sortSettingToAction: (sortSetting: Types.SortSetting) => () => {
               dispatch(FsGen.createSortSetting({path, sortSetting: Constants.makeSortSetting(sortSetting)}))
-              if (evt) {
-                dispatch(navigateUp())
-                evt.stopPropagation()
```

Comment:
does removing this break anything?

Reply:
I clicked around and nothing broke. I don't remember why we had this. Do you remember?

Reply:
Nope, ship it.

Reply:
In fact, evt is always falsey here, and this doesn't matter anymore. So it could be this line was from when we had some old version of PopupMenu which needed that.

Reply:
Cool